tutorials/01_manual_input.tex

```diff
@@ -38,108 +38,94 @@ \subsection*{Create a new workspace}
+\noindent \faGraduationCap \, For each criterion, there can be one or more references from which summary statistics or treatment effect estimates are taken. A reference can be a single pivotal study but could also be a pooled analysis, such as a combined safety pool or a (network-) meta-analysis. Its origins are specified via its name (the ‘Reference’ field) and optionally its URL.
 \newline
 
-\noindent \leftpointright \, On the HbA1c criterion, click the ‘Add data source’ button. In the dialog, specify the reference “Pooled data from EFC6014 and EFC10743” and leave the ‘Reference URL’ field empty. Click the ‘Add’ button. You now should have one data source for the HbA1c criterion. Click the ‘Add data source’ button and add the second data source woth the reference “EFC6015” while leaving the ‘Reference URL’ field empty again. Then click ‘Add.’ Now there should be a second row in the data sources list for the criterion.
+\noindent \leftpointright \, For the HbA1c criterion, edit the existing reference, by clicking on the last cell in the row, and fill in the dialog to be  “Pooled data from EFC6014 and EFC10743”, leaving the ‘Reference URL’ field empty. After closing the dialog, the reference should now display your changes. Click  the ‘Add reference’ button (the '+' symbol) just below this row. A second reference row will appear. Edit this one so the reference is “EFC6015”, again leaving the ‘Reference URL’ field empty.
 
 \subsection*{Add nausea as an unfavourable effect}
-\noindent \leftpointright \, Click the ‘Add criterion’ button, fill in the dialog as specified in Figure \ref{fig:crit2}, and click ‘Add’ (note that favorability is now ‘Not favorable’).
+\noindent \leftpointright \, Edit 'criterion 2' to be called 'Nausea' and set its description to 'Incidence of nausea'. Then, change it from a favorable to an unfavorable effect by clicking the icon on the left of the row, with the thumb pointing down.
 \newline
 
-\begin{figure}[!h]
-    \centering
-	\includegraphics[width=\textwidth]{fig/addNausea.png}
-    \caption{Add criterion dialog for Nausea.}
-	\label{fig:crit2}
-\end{figure}
-
-\noindent \leftpointright \, Click the ‘Add data source button and enter “Pooled data from all phase 2/3 controlled studies” as the the reference for this data source.  There are no further data sources for this criterion.
+\noindent \leftpointright \, Change the reference to “Pooled data from all phase 2/3 controlled studies”.  There are no further references for this criterion.
 
 \subsection*{Add lixisenatide and placebo as alternatives}
-\noindent \leftpointright \, Under the ‘Alternatives’ header, click the ‘Add Alternative’ button and fill in “lixisenatide” in the ‘Title’ text box. Click the ‘Add’ button. Repeat the process and add a “placebo” alternative.
+\noindent \leftpointright \, Rename 'alternative 1' to by clicking on it and entering 'lixisenatide' in the 'Title' text box. Rename 'alternative 1' to 'placebo'.
```

Comment:
Delete 'to' before by, no more, no less. 'To' shall be the word thy shalt delete.